src/front/js/pages/parents.js

```diff
@@ -0,0 +1,78 @@
+import React from "react";
+import "../../styles/parents.css";
+import Form from "react-bootstrap/Form";
+import Button from "react-bootstrap/Button";
+import Dropdown from "react-bootstrap/Dropdown";
+import Col from "react-bootstrap/Col";
+import Container from "react-bootstrap/Container";
+import Row from "react-bootstrap/Row";
+
+import "bootstrap/dist/css/bootstrap.min.css";
+
+const Parents = () => {
+  return (
+    <>
+      <div className="container-fluid">
+        <Container>
+          <Row>
+            <Col className="col-2">
+              <img
+                src="https://previews.123rf.com/images/rosinka79/rosinka791704/rosinka79170400108/77286091-tambov-f%C3%A9d%C3%A9ration-de-russie-21-septembre-2016-portrait-de-la-famille-lego-figurines-d-un-p%C3%A8re-d-une-.jpg"
+                id="profile"
+              ></img>
+            </Col>
+            <Col className="col-5">
+              <input placeholder="Add your description" id="bio"></input>
```

Comment:
I *still* think you didn't want to use this tag here, Maise. Why don't you try <textarea> and tell me what you think?
```suggestion
              <textarea placeholder="Add your description" id="bio" />
```